### PR TITLE
domd: tune weston.ini and weston-seats.rules...

### PR DESCRIPTION
**Pull request by iartemenko**
in order to 'kingfisher' needs.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend [hunk 1]
```diff
@@ -13,13 +13,20 @@ python __anonymous () {
 }
 
 SRC_URI_append = "file://weston-seats.rules \
+                  file://weston-seats-kf.rules \
 "
 
 FILES_${PN} += " \
     ${sysconfdir}/udev/rules.d/weston-seats.rules \
+    ${sysconfdir}/udev/rules.d/weston-seats-kf.rules \
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend]: this is not needed as we do not install "*kf*" version of the file

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DEVPATH=="/devices/platform/soc/ee000000.usb/usb5/5-1/5-1.1/*", ENV{WL_OUTPUT}="HDMI-A-1"
+DEVPATH=="/devices/platform/soc/ee000000.usb/usb5/5-1/5-1.2/*", ENV{WL_OUTPUT}="HDMI-A-2"
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules]: how about something more generic?
```
DEVPATH=="/devices/platform/soc/ee00*.usb/usb*/*/*1/*", ENV{WL_OUTPUT}="HDMI-A-1"
DEVPATH=="/devices/platform/soc/ee00*.usb/usb*/*/*2/*", ENV{WL_OUTPUT}="HDMI-A-2"

```

Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules]: Done

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend (hunk 1 above)
Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend]: Done

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules (hunk 2 above)
Comment (andr2000) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules]: we need this to be tested on the target

Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston/weston-seats-kf.rules]: Tested, it works.

**Pull request merged**